utilities/decrypt_data/decrypt_object.py

```
@@ -0,0 +1,112 @@
+
+    except Exception as e:
+        logger.error("Decrypt API error for %r: %s", payload, e)
+        return {}
+
+
+def migrate_jsonb():
+    table = CONFIG["table"]
+    id_col = CONFIG["id_column"]
+    jcol   = CONFIG["json_column"]
+
+    select_sql = f"SELECT {id_col}, {jcol} FROM {table} WHERE {jcol} IS NOT NULL"
```

Ruff (0.11.9)

Possible SQL injection vector through string-based query construction

(S608)